crates/bevy_diagnostic/src/log_diagnostics_plugin.rs

```diff
@@ -69,6 +69,36 @@ impl LogDiagnosticsPlugin {
                     );
                     return;
                 }
+
+                if let Some(min) = diagnostic.min() {
+                    info!(
+                        target: "bevy diagnostic",
+                        // Suffix is only used for 's' as in seconds currently,
+                        // so we reserve one column for it; however,
+                        // Do not reserve one column for the suffix in the average
+                        // The ) hugging the value is more aesthetically pleasing
+                        "{name:<name_width$}: {value:>11.6}{suffix:1} (min {min:>.6}{suffix:})",
+                        name = diagnostic.name,
+                        suffix = diagnostic.suffix,
+                        name_width = crate::MAX_DIAGNOSTIC_NAME_WIDTH,
+                    );
+                    return;
+                }
+
+                if let Some(max) = diagnostic.max() {
+                    info!(
+                        target: "bevy diagnostic",
+                        // Suffix is only used for 's' as in seconds currently,
+                        // so we reserve one column for it; however,
+                        // Do not reserve one column for the suffix in the average
+                        // The ) hugging the value is more aesthetically pleasing
+                        "{name:<name_width$}: {value:>11.6}{suffix:1} (max {max:>.6}{suffix:})",
+                        name = diagnostic.name,
+                        suffix = diagnostic.suffix,
+                        name_width = crate::MAX_DIAGNOSTIC_NAME_WIDTH,
+                    );
+                    return;
+                }
```

Comment:
Before this added code there is a block  `if let Some(average) = diagnostics.average() {` that contains a `return` statement, so the added code will probably  never be reached.
On top of that, this kind of code would print 3 log lines one with 'value + avg' , one with 'value + min' and one with 'value + max'

If we want to include, min and max in the log diagnostics, I think we need to include them in the parenthesized part `(avg XX, min YY, max ZZ)`